db/migrate/20200723203155_change_mentions_notes_podcasts_roles_related_ids_to_bigints.rb

```diff
@@ -0,0 +1,41 @@
+class ChangeMentionsNotesPodcastsRolesRelatedIdsToBigints < ActiveRecord::Migration[6.0]
+  def up
+    ActiveRecord::Base.connection.execute("DROP VIEW IF EXISTS hypershield.mentions")
+    safety_assured { change_column :mentions, :user_id, :bigint }
+
+    ActiveRecord::Base.connection.execute("DROP VIEW IF EXISTS hypershield.notes")
+    ActiveRecord::Base.connection.execute(
```

Comment:
Somewhat bizarrely Ruby also accepts the following way of formatting heredocs as arguments:

```ruby
ActiveRecord::Base.connection.execute(<<-SQL)
  ALTER TABLE notes
    ALTER COLUMN author_id TYPE bigint,
    ALTER COLUMN noteable_id TYPE bigint
SQL

#<PG::Result:0x00000001154e5320 status=PGRES_COMMAND_OK ntuples=0 nfields=0 cmd_tuples=0>
```

It's always been my preferred form since I find it reads cleaner. 

No need to change was only meant as an FYI.